css/components/card.css

```diff
@@ -1,77 +1,196 @@
-.card-grid {
-	display: grid;
-	gap: 1rem;
+/* .card-grid {
+  display: grid;
+  gap: 1rem;
 }
 
 .card-row {
-	grid-template-columns: 1fr;
+  grid-template-columns: 1fr;
 }
 
 .card {
-	display: flex;
-	flex-direction: column;
-	justify-content: space-between;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
 .card-horizontal {
-	display: grid;
-	grid-template-columns: repeat(2, 1fr);
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
 }
 
 .card-bg {
-	background-color: var(--bg-500);
+  background-color: hsl(var(--bg-500));
 }
 
 .card-border {
-	border: 1px solid var(--bg-500);
+  border: 1px solid hsl(var(--bg-500));
 }
 
 .card-text-center {
-	text-align: center;
+  text-align: center;
 }
 
 .card-title {
-	color: var(--font-700);
+  color: hsl(var(--font-700));
 }
 
 .card-p {
-	padding: 0.5;
+  padding: 0.5;
 }
 
 .card-p-t-b {
-	padding: 2rem 1rem;
+  padding: 2rem 1rem;
 }
 
 .card-p-eq-1 {
-	padding: 1rem;
+  padding: 1rem;
 }
 
 .card-p-eq-2 {
-	padding: 2rem;
+  padding: 2rem;
 }
 
 .card-image {
-	inset: 0 0;
+  inset: 0 0;
 }
 
 .card-text {
-	padding: 0 0.5rem;
+  padding: 0 0.5rem;
 }
 
 .image-center {
-	display: flex;
-	align-items: center;
-	justify-content: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
same as above

Reply:
I have seprate utility classes in https://github.com/ttahm3d/chunks/blob/dev/css/components/layout.css. I don't want user to enter another class.